index.html

```diff
@@ -1433,20 +1433,16 @@ <h3>Extensibility</h3>
         </p>
 
         <p>
-An extensibility model that is this dynamic does increase implementation burden.
-Software written for such a system will have to determine if relying on
-<a>verifiable credentials</a> with extensions is acceptable based on the
-risk profile of the application. Some applications may choose to only process
-certain extensions while highly secure environments may require that no
-extensions are allowed. These decisions are up to the developers of these
+A dynamic extensibility model such as this does increase the implementation
+burden. Software written for such a system has to determine whether
+<a>verifiable credentials</a> with extensions are acceptable based on the
+risk profile of the application. Some applications might accept only
+certain extensions while highly secure environments might not accept any
+extensions. These decisions are up to the developers of these
 applications and are specifically not the domain of this specification.
         </p>
 
-        <p>
-Applications that do not understand the semantic meaning of all properties
-while processing a <a>verifiable credential</a> or a
-<a>verifiable presentation</a> MUST produce an error.
-        </p>
+
```

Comment:
Software that does not understand the semantic meaning of all the properties in a <a>verifiable credential</a> or a
<a>verifiable presentation</a> MUST fail to verify the data structure.

Reply:
Your proposed language makes verification dependent on the software rather than on conformance. See https://github.com/w3c/vc-data-model/issues/317

Reply:
Accepted. You are good to go

Reply:
This is now resolved